### PR TITLE
test: standard cron syntax

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: docs/content/Schema/cube.md [hunk 1]
```diff
@@ -300,18 +300,25 @@ cube(`OrderFacts`, {
 });
 ```
 
-`every` can accept only equal time intervals - so  "Day of month" and "month" intervals in CRON expressions are not supported.
+`every` can accept only equal time intervals - so "Day of month" and "month" intervals in CRON expressions are not supported.
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| Cube.js supports two different formats of CRON expressions: with leading seconds (6 parts) and leading minutes (5 parts).
```
Comment (hassankhan) [inline docs/content/Schema/cube.md:307]: @ovr any chance we could add an example for both formats, please? 😀

**Review by hassankhan — APPROVED** [review]

File: docs/content/Schema/cube.md (hunk 1 above)
Comment (ovr) [reply docs/content/Schema/cube.md:307]: @hassankhan shall I add it to this warning or below?

Comment (hassankhan) [reply docs/content/Schema/cube.md:307]: I think to the warning would be best; something like:

```
Cube.js supports two different formats of CRON expressions; with leading seconds (EXAMPLE_HERE) and with leading minutes (EXAMPLE_HERE).
```

**Review by hassankhan — APPROVED** [review]
LGTM 🙌

**Pull request merged**